app/untitled.sc

```diff
@@ -0,0 +1,9 @@
+object Error
+
+case class Income(rent: Option[BigDecimal])
+
+case class Expense(travel: Option[BigDecimal])
+
+case class Object(income: Option[Income], expense: Option[Expense])
+
+val allEmpty = Object
```

Comment:
This file is called "untitled.sc"

Reply:
I'm also unsure if it's actually being used anywhere?

Reply:
It is not.